ingress/controllers/nginx/README.md

```diff
@@ -102,6 +104,7 @@ Check NGINX it is running with the defined Ingress rules:
 $ LBIP=$(kubectl get node `kubectl get po -l name=nginx-ingress-lb --template '{{range .items}}{{.spec.nodeName}}{{end}}'` --template '{{range $i, $n := .status.addresses}}{{if eq $n.type "ExternalIP"}}{{$n.address}}{{end}}{{end}}')
 $ curl $LBIP/foo -H 'Host: foo.bar.com'
 ```
+You may need to add a firewall rule to allow traffic on port 80. This is related to your cloud provider.
```

Comment:
Oh, I see you have it below. 

Hmm, how about just creating a different section:

## Running multiple ingress controllers 

If you're running multiple ingress controllers, or running on a cloudprovider that natively handles 
ingress, you need to specify the annotation `kubernetes.io/ingress.class: "nginx"` in all ingresses 
that you would like this controller to claim. Not specifying the annotation will lead to multiple 
ingress controllers claiming the same ingress. Specifying the wrong value will result in all ingress 
controllers ignoring the ingress. Multiple ingress controllers running in the same cluster was not 
supported in Kubernetes versions < 1.3.

## Running on Cloudproviders

If you're running this ingress controller on a cloudprovider, you should assume the provider also has a native 
Ingress controller and specify the `ingress.class` annotation as indicated in [this section](#running-multiple-ingress-controllers).
In addition to this, you will need to add a firewall rule for each port this controller is listening on, i.e :80 and :443.